js/src/collections/search_collection.js

```diff
@@ -0,0 +1,26 @@
+var _ = require('lodash');
```

Comment:
I'm wondering if we need a `SearchCollection` at all, because right now it does the same thing as the `CountriesCollection`. We're not certain that the search will do more than just search for countries in the future, so I would just re-use the countries collection within the search so we have less code to maintain. What do you think?